mlrun/datastore/google_cloud_storage.py

```diff
@@ -59,12 +59,12 @@ def get_storage_options(self):
             except json.JSONDecodeError:
                 # If it's not json, handle it as a filename
                 token = credentials
-            return dict(token=token)
+                return self._sanitize_storage_options(dict(token=token))
         else:
             logger.info(
                 "No GCS credentials available - auth will rely on auto-discovery of credentials"
             )
-            return None
+            return self._sanitize_storage_options(None)
```

Comment:
Here as well, simpler to just return `{}`.

Reply:
same is here